src/app/modules/group/components/joined-group-list/joined-group-list.component.scss

```diff
@@ -0,0 +1,125 @@
+@import "src/colors.scss";
+@import "src/geometry.scss";
+
+:host ::ng-deep .ui-table .ui-table-thead > tr > th {
+  // position: -webkit-sticky;
+  // position: sticky !important;
+  // top: 5.8333rem;
+  // overflow: visible;
+}
+
+@media screen and (max-width: 64em) {
+  :host ::ng-deep .ui-table .ui-table-thead > tr > th {
+    top: 8.3333rem;
+  }
+}
+
+:host {
+  position: relative;
+  display: block;
+
+  ::ng-deep .ui-table {
+    margin-bottom: 1rem;
+    font-family: 'titillium', sans-serif;
+
+
+    .ui-table-thead > tr > th {
```

Comment:
When the ngprime10 will be merge, you 'll need to rebase and convert all ui-table-* to p-datatable-*

Reply:
fixed